site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -36,25 +46,22 @@ The folder structure is mostly based on the [Cypress scaffold](https://docs.cypr
   |-- package.json
 ```
 
-- `/e2e/cypress/fixtures` or [Fixture Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files):
+1. `/e2e/cypress/fixtures` or [Fixture Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files):
     - Fixtures are used as external pieces of static data that can be used by tests.
     - Typically used with the `cy.fixture()` command and most often when stubbing Network Requests.
-- `/e2e/cypress/integration` or [Test Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files)
+2. `/e2e/cypress/integration` or [Test Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files)
     - To start writing tests,
         - simply create a new file (e.g. `login_spec.js`) within `/e2e/cypress/integration` folder and;
         - refresh tests list in the Cypress Test Runner and a new file should have appeared in the list.
     - Subfolder naming convention depends on test grouping, which is usually based on the name of tab in the Release Testing spreadsheet (e.g. `/e2e/cypress/integration/messaging/` for "Messaging" tab).
     - Test cases that require an enterprise license should fall under `/e2e/cypress/integration/enterprise/`. This is to easily identify license requirements, both during local development and production testing for enterprise features.
-- `/e2e/cypress/plugins` or [Plugin Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Plugin-files)
+3. `/e2e/cypress/plugins` or [Plugin Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Plugin-files)
     - A convenience mechanism that automatically include the plugins before running every single spec file.
-- `/e2e/cypress/support` or [Support Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Support-file)
+4. `/e2e/cypress/support` or [Support Files](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Support-file)
     - The support file is a place to put reusable behaviour such as Custom Commands or global overrides that are wished to be applied and available to all of spec files.
-- `/e2e/cypress/utils`
-    - A place for common utility functions
-- `/e2e/cypress.json`
-    - Use to store Cypress [configuration](https://docs.cypress.io/guides/references/configuration.html#Options)
-- `/e2e/package.json`
-    - Use to store all dependencies related to Cypress end-to-end testing
+5. `/e2e/cypress/utils` for common utility functions.
+6. `/e2e/cypress.json` for Cypress [configuration](https://docs.cypress.io/guides/references/configuration.html#Options).
+7. `/e2e/package.json` for all dependencies related to Cypress end-to-end testing.
 
 ## What requires an E2E Test?
```

Comment:
(nice-to-have) Linking to a PR or a ticket that gives an example for each of these use cases would be nice to have. Some like features and bugs are self-explanatory, but "Cases that are not covered by unit or integration tests" may not be.

Reply:
Thanks @jasonblais! Links and sample case added. Also replaced the last item with test cases from TM4J.
@justinegeffen I pushed an updates per comment. Please take a look again, thanks!

<img width="791" alt="Screen Shot 2020-07-21 at 12 57 25 AM" src="https://user-images.githubusercontent.com/5334504/87964928-87a4b600-caed-11ea-9d29-9c4d4327a43b.png">